### ISSUE TITLE
sundials+rocm spack test fails with no output

### ISSUE THREAD

**Issue by wspear** [E4S-Project/testsuite#60]
@balos1  @cswoodward  @gardner48

On Crusher the internal spack test for sundials works for the cpu build but the +rocm build (variant below) fails with an empty log output. Assuming this variant is valid, is it possible the sundials test could be updated to support gpu variants? 

```
-- linux-sles15-zen3 / gcc@11.2.0 -------------------------------
e5jbpyc sundials@6.4.1+ARKODE+CVODE+CVODES+IDA+IDAS+KINSOL~cuda+examples+examples-install~f2003~fcmix+generic-math~ginkgo~hypre~int64~ipo~klu~kokkos~kokkos-kernels~lapack~magma~monitoring+mpi~openmp~petsc~profiling~pthread~raja+rocm+shared+static~superlu-dist~superlu-mt~sycl~trilinos amdgpu_target=gfx90a build_system=cmake build_type=RelWithDebInfo cstd=99 cxxstd=14 logging-level=0 logging-mpi=OFF precision=double

```

**Comment by gardner48** [comment]
@wspear This might be due to a typo fixed in spack/spack#34539. Could you retry and let me know if the issues persists?

**Comment by wspear** [comment]
It looks like that did the trick. Thanks!
